### PR TITLE
use absolute path for composite

### LINKED ISSUE #16: Hardcoded path breaks Buildship
In /xtext-umbrella/gradle-composite/build.gradle there are hardcoded path entries to `../xtext-umbrella/releng/`

`xtext-umbrella` is not the project name but the git root directory which can be different depending on user configuration used during the Ommph setup.

Buildship stops resolution with:
```
Caused by: org.gradle.api.InvalidUserDataException: Project directory '/Users/dhuebner/git/xtext-umbrella/releng/gradle-composite' does not exist.
	at org.gradle.initialization.ProjectDirectoryProjectSpec.checkPreconditions(ProjectDirectoryProjectSpec.java:52)
	at org.gradle.initialization.AbstractProjectSpec.containsProject(AbstractProjectSpec.java:27)
	at org.gradle.initialization.DefaultSettingsLoader.findAndLoadSettings(DefaultSettingsLoader.java:51)
	at org.gradle.internal.composite.CompositeBuildSettingsLoader.findAndLoadSettings(CompositeBuildSettingsLoader.java:48)
	at org.gradle.initialization.NotifyingSettingsLoader.findAndLoadSettings(NotifyingSettingsLoader.java:33)
	at org.gradle.initialization.DefaultGradleLauncher.doBuildStages(DefaultGradleLauncher.java:147)
	at org.gradle.initialization.DefaultGradleLauncher.doBuild(DefaultGradleLauncher.java:119)
	... 44 more```

### PR DESCRIPTION
**Pull request by cdietrich**
use absolute path for composite. fixes #16

Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by cdietrich** [comment]
please review.

File: releng/gradle-composite/build.gradle [hunk 1]
```diff
@@ -8,15 +9,15 @@ task('generatePrefs') {
...
-            (new File(file, '.settings/org.eclipse.buildship.core.prefs')).text = "connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)\nconnection.project.dir=../xtext-umbrella/releng/gradle-composite\nproject.path=\\:"+file.name+"\n"
+            (new File(file, '.settings/org.eclipse.buildship.core.prefs')).text = "connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)\nconnection.project.dir="+base+"\nproject.path=\\:"+file.name+"\n"
                     
             file.listFiles().each { File fx -> 
                 if (fx.isDirectory() && new File(fx, "build.gradle").exists()) {
                     println  "generating buildship prefs for " + fx.path
                     if (!(new File(fx, '.settings')).exists()) {
                         new File(fx, '.settings').mkdir();
                     }
-                    (new File(fx, '.settings/org.eclipse.buildship.core.prefs')).text = "connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)\nconnection.project.dir=../../xtext-umbrella/releng/gradle-composite\nproject.path=\\:"+fx.name+"\n"
```
Comment (spoenemann) [inline releng/gradle-composite/build.gradle]: The default setting for Oomph seems to name the repository `xtext-umbrella-master`

Comment (cdietrich) [inline releng/gradle-composite/build.gradle]: sure @kthoms do you have an idea?

Comment (cdietrich) [inline releng/gradle-composite/build.gradle]: hmm i just gave it a try on a linux vm and it just checks out as xtext-umbella

Comment (dhuebner) [inline releng/gradle-composite/build.gradle]: It's not a linux issue but Oomph user specific configuration <repo-name>-<branch>. We need just read the folder name and not hardcode it.  Is it a problem?

Comment (cdietrich) [inline releng/gradle-composite/build.gradle]: no its not, this is why i ask you to test this

Comment (cdietrich) [inline releng/gradle-composite/build.gradle]: or i merge my pull request and you test then

**Comment by cdietrich** [comment]
so is it ok to merge?

File: releng/gradle-composite/build.gradle (hunk 1 above)
Comment (dhuebner) [inline releng/gradle-composite/build.gradle]: Let's give it a try

**Comment by kthoms** [comment]
@miro the checkout location depends in the git clone location rule, this is user dependent.

**Comment by cdietrich** [comment]
cant we simply prevent this?

**Comment by kthoms** [comment]
I don't think so.

**Pull request merged**